ci/generate-workflows.py

```diff
@@ -845,7 +848,8 @@ def prep_environment(self, cache=True):
 
         steps += self.install_openssh_server()
         steps += self.checkout()
-        steps += self.install_rust(cache="mac" not in self.name)
+        # We should be able to cache mac builds now?
+        steps += self.install_rust() # cache="mac" not in self.name)
```

Comment:
`sccache` might fix macos caching or it might break because of this (it is necessary because otherwise `RUSTC_WRAPPER` is set but `sccache` is not installed).

I might be able to conditionally disable the env vars, but it is worth a try.